### PR TITLE
Created styles; Added styles to index

### PR DESCRIPTION
**Pull request by rromanenko**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <html>
-<head>
+<>
```
Comment (nikita-rogatnev) [inline index.html]: Typo, fix this

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+body {
+background:red;
```
Comment (nikita-rogatnev) [inline css/styles.css]: please use tabulation

File: index.html (hunk 1 above)
Comment (rromanenko) [reply index.html]: Fixed

**Comment by nikita-rogatnev** [comment]
Good job

File: css/styles.css (hunk 2 above)
Comment (rromanenko) [reply css/styles.css]: Fixed

**Pull request merged**